applications/rag/frontend/variables.tf

```diff
@@ -66,4 +66,86 @@ variable "google_service_account" {
   type        = string
   description = "Google Service Account name"
   default = "frontend-gcp-sa"
+}
+
+variable "add_auth" {
+  type        = bool
+  description = "Enable iap authentication on frontend"
+  default     = true
+}
+
+variable "k8s_ingress_name" {
+  type    = string
+  default = "frontend-ingress"
+}
+
+variable "k8s_managed_cert_name" {
+  type          = string
+  description   = "Name for frontend managed certificate"
+  default       = "frontend-managed-cert"
+}
+
+variable "k8s_iap_secret_name" {
+  type    = string
+  default = "frontend-secret"
+}
+
+variable "k8s_backend_config_name" {
+  type        = string
+  description = "Name of the Backend Config on GCP"
+  default     = "frontend-iap-config"
+}
+
+variable "k8s_backend_service_name" {
+  type        = string
+  description = "Name of the K8s Backend Service, this is defined by Frontend"
+  default     = "rag-frontend"
+}
+
+variable "k8s_backend_service_port" {
+  type        = number
+  description = "Name of the K8s Backend Service Port"
+  default     = 8080
+}
+
+variable "brand" {
+  type        = string
+  description = "name of the brand if there isn't already on the project. If there is already a brand for your project, please leave it blank and empty"
+  default     = ""
```

Comment:
FYI, if the brand var is empty.. we also attempt to create a brand..

Reply:
Yeah, the brand will be set in rag